client/src/views/Knowledge/components/Graphs/Grafer.vue

```diff
@@ -29,24 +46,54 @@
     @Prop({ default: null })
     private layer: string;
 
-    public mounted (): void {
-      this.loadGraph().then(data => {
-        this.controller = new GraferController(this.$refs.canvas as HTMLCanvasElement, data);
-        this.forwardEvents(this.controller);
-        this.loading = false;
-      });
-    }
+    // Initialize as undefined to prevent Vue from observing changes within these large datasets
+    // Grafer data is stored as they are required for mapping bgraph queries to grafer layers
+    graferNodesData: any = undefined;
 
-    // temporary demo functions
-    async loadGraph (): Promise<GraferControllerData> {
-      const points = {
-        data: await loadJSONLFile(`/grafer/${this.model}/points.jsonl`),
-      };
+    public async mounted (): Promise<void> {
+      const [
+        nodesFile,
+        nodeAttsFile,
+        nodeLayoutFile,
+        groupsFile,
+      ] = await Promise.all([
+        // getS3Util('research/KB/dist/kaggle/v4.0_citations_small/nodes.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citations_small/nodeAtts.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citations_small/nodeLayout.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citations_small/groups.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citation/nodes.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citation/nodeAtts.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citation/nodeLayout.jsonl'),
+        // getS3Util('research/KB/dist/kaggle/v4.0_citation/groups.jsonl'),
+        // getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_top2vec/nodes.jsonl'),
+        // getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_top2vec/nodeAtts.jsonl'),
+        // getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_top2vec/nodeLayout.jsonl'),
+        // getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_top2vec/groups.jsonl'),
+        getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_nonTop2Vec/nodes.jsonl'),
+        getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_nonTop2Vec/nodeAtts.jsonl'),
+        getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_nonTop2Vec/nodeLayout.jsonl'),
+        getS3Util('research/KB/dist/wisconsin/xdd-covid-19-8Dec-doc2vec/v4.0_nonTop2Vec/groups.jsonl'),
+      ]);
 
-      const colors = CLUSTER_GRAPH_COLORS;
-      const layers = await this.loadModelLayers(points);
+      const info: LayoutInfo = {
```

Comment:
LayoutInfo object contains fields that are are not being used downstream. For examples 'No file selected.' is a remnant of the grafer interface's drag and drop file prompt.